docs/operations/runbooks/rotate_application_secrets.md

```diff
@@ -27,7 +27,7 @@ To rotate secrets, create a new `credentials-<ENVIRONMENT>.json` file, upload it
 Example:
 
 ```bash
-cf uups getgov-credentials -p credentials-unstable.json
+cf cups getgov-credentials -p credentials-unstable.json
```

Comment:
*suggestion (non-blocking)*: I slightly prefer the long form of the `create-user-provided-service` command for the purpose of documentation.